index.html

```diff
@@ -190,7 +229,7 @@ <h1 class="program__heading--text">Main Program</h1>
             <img src="img/benkler.png" alt="Author" class="program__img" />
           </div>
           <div class="program__text">
-            <h1 class="program__name">Yochai Benkler</h1>
+            <h2 class="program__name">Yochai Benkler</h2>
```

Comment:
As suggested by the previous reviewer and as required for this project, kindly ensure that you load all the featured speakers dynamically from JavaScript. You do not need to have any of them cards hard-coded into your HTML